added uwinsite guide (#75)
UWindsorCSS/wiki: resources/guides/uwinsite_guide.md

---
id: uwinsite_guide
title: UWinsite Guide
sidebar_label: UWinsite Guide
slug: /guides/uwinsite_guide
---

> 

Reviewer: Please include a `_Last updated on ..._` . You can read the specific format that we want in [REQUIREMENTS.md](https://github.com/UWindsorCSS/wiki/blob/master/REQUIREMENTS.md)

This is a guide on how to use and access the important sections on your UWinsite Student page.

## Course Management

### How to Plan & Enroll in Classes

To plan your courses and register in classes in UWinsite Student, you will use the
**Manage Classes** section on the Student Homepage.

#### Planning your Classes

You can start planning your classes and adding them to your Shopping Cart in UWinsite
Student when the validation period opens. The validation period is when the timetable
for the next semester is close to final and the courses being offered are visible in
UWinsite Student. Typically, the validation period starts two to three weeks before
registration begins.

The preferred method for placing your classes into the Shopping Cart is through Class
Search and Enroll. This method shows you just the courses that are offered during the
term for which you are planning.

**To get started planning your classes:**

> ![img](../../static/img/uwinsite_guide/uwinsite_homepage.jpg)

Reviewer: the alt text(the part inside the `[..]`) should always be a title or description of the image. it is used by screen readers to help people who cannot see the image.

for example if I were to put a photo of a chair, I would write
```md
![photo of chair](../../static/img/chair.jpg)
```

BTW this goes for all images without alt text in this PR(and in general)

**Class Selection**, you can view the time, date, instructor, location and available seats
for each section of the class.
![img](../../static/img/uwinsite_guide/course_info_and_class_selection.jpg)
7. Select a class section. You can click anywhere on the class except for the Lecture or
Lab link to select it.
8. You’ll now be directed to the confirmation process:
    1. Review **Class Selection** and click **Next**
    2. Review **Class Preferences**
        + This is where you have the option to add yourself to the waitlist if the class is
        full. Select **Yes** if this will not put you in course overload (*normally 5 classes, but
        check your program*). 
        + Select **No** if having this course on the waitlist would put you in course overload.
        You will instead need to use **Swap Classes** so UWinsite Student knows what
        course to drop if you are to reach the top of the waitlist.
    3. Click **Accept**
    4. On the **Enroll** or **Add to Cart** page, click the **Add to Shopping Cart** button and
    then click **Next**.
    5. On the **Review and Submit** page, review your selections and then click **Submit**.

#### Registering in your Classes

> Registration at UWindsor is by appointment time. Registration appointment times are
> assigned based on program and level of study. Your registration appointment time is

Reviewer: ```suggestion
Registration at UWindsor is by appointment time. Registration appointment times are
assigned based a number of factors including program and level of study. Your registration appointment time is
```

I think this is better because its not based solely program and year.

when you can start to register. It is important to register early to have a better chance
of getting into the classes that you want to take. Registration does, however, remain
open until the tenth day of classes so you may make changes to your courses so long
as there is space.

Once registration appointment times are assigned for a term, you can find yours under
**My Academics -> Enrolment Deadlines**. Your registration appointment time will also be
sent to your UWindsor email account.

**When your registration appointment time opens:**

1. On the Student Homepage, click the **Manage Classes** tile.
2. Click the **Shopping Cart** tab in the left navigation menu.
3. Click the radio button next to each of your preferred classes and then click the **Enroll** button.
    + If you are successfully enrolled in a class, you will see a **green check mark** beside it.
    + If you are not enrolled in a class, you will see a **red X** beside it. 

> 4. There are several reasons why you might not be successfully enrolled in a class:
>     + The class is full.
>     + Enrolment in the class is restricted for various reasons. For example: it is reserved
>     for students in particular majors or levels of study.
>     + You may be missing a prerequisite (this should not be the case IF you validated
>     your classes).

Reviewer: ```suggestion
4. There are several reasons why you might not be successfully enrolled in a class:
    + The class is full.
    + There is a time conflict.
    + Enrolment in the class is restricted for various reasons. For example: it is reserved
    for students in particular majors or levels of study.
    + You may be missing a prerequisite (this should not be the case IF you validated
    your classes).
```

Sometimes course times change to be conflicted or were always conflicting and the validator didnt catch it. I think its important to mention that case.

#### Enrollment Deadlines

Here, you can view your appointments and enrollment dates for another term including the session, when the appointment starts, and the last date to enroll.

#### Change of Program Application

Here, current students can change their programs and this is also for students who applied for part-time studies who wish to apply for full-time studies. You'll also see the deadlines of the application for each term.

To start a change of program application: 
1. Click **Create New Application**. You will see a variety of areas that need to be completed. 
2. Now, click the **Application Start** button which you may need to scroll down to see. 
![img](../../static/img/uwinsite_guide/change_program_app.jpg)
3. Then, complete the required fields on the page. 
4. Once you complete a page of your application, click the **Save** button to save what you've filled 
5. Click the **Continue** button to proceed to the next page and do the same until all pages have been filled.
6. When you complete your application, click **Submit Application** 

> **Note**: *Make sure you save your application often. You can also save what you've currently filled and come back another time to complete the rest of your application.*

Reviewer: this would be better as a [docusaurus admonition](https://docusaurus.io/docs/markdown-features/admonitions)

I think this goes for any note in this document

#### Academic Progress Summary

This tab shows you the graphical representation of your academic progress. 

#### Academic Progress

This tab gives you an in-depth look at your academic progress. Academic Progess breaks down your program requirements. You will be able to view bar charts showing the completion level of your program requirements.

#### What-if Report

The What-if Report allows you to view **what requirements you need to fulfill if you decided to switch programs**. 

To run a What-if Report: 
1. Click **Create New Report** in the **View What-if Report** tab. 
![img](../../static/img/uwinsite_guide/what_if_report.jpg)
2. Then you choose your **Career** followed by your **Academic Program**, then your **Area of Study**, and then your **Concentration**. 
3. After that, click **Browse Course Catalog**. 
4. Then select **Subject Codes** for inclusion in your course what-if report. 
5. Now, click **Submit Request**. 

The system will generate a response based on your choices. You will be provided with what course will be required and what courses do not fulfill your academic program requirements. 

### Academic Records

#### Course History

> Course History alows you to view the courses you have completed, transferred from, are planning to take, and that are currently in progress. You can see the **status** of each of your courses on the right hand side.

Reviewer: ```suggestion
Course History allows you to view the courses you have completed, transferred from, are planning to take, and that are currently in progress. You can see the **status** of each of your courses on the right hand side.
```

#### View Grades

Here, you can look at your grades by semester. Choose what semester you would like to view and you'll see a listing of the courses you took including the grades you recieved and the units/credits the course is worth. Note: **units = credits**. 

An undergraduate student also has an option to **submit a request to appeal a grade**. 
If you are not satisfied with a grade you receive in any course, you should first discuss it with the **individual instructor** who assigned the grade. You should make every effort to resolve the issue with the instructor by outlining the reasons for your dissatisfaction and attempt to come to a mutually agreeable resolution.

> If you are unable to resolve the problem with your instructor, you may consider formally appealing your grade based on **Incorrect Evaluation** or **Procedural Irregularity**. Students who wish to appeal a grade in a course must submit a Grade Appeal through UWinsite Student no later than three weeks after the final grade has been posted. A letter of rationale and supporting documentation must accompany the appeal form along with an administrative fee of $20 (this fee will be refunded if your appeal is successful). An appeal for compassionate reasons do not have an administrative fee of $20. 

Reviewer: ```suggestion
If you are unable to resolve the problem with your instructor, you may consider formally appealing your grade based on **Incorrect Evaluation** or **Procedural Irregularity**. Students who wish to appeal a grade in a course must submit a Grade Appeal through UWinsite Student **no later than three weeks after the final grade has been posted**. A letter of rationale and supporting documentation must accompany the appeal form along with an administrative fee of $20 (this fee will be refunded if your appeal is successful). An appeal for compassionate reasons do not have an administrative fee of $20. 
```

I think that is important and should be bolded

> There are twp types of appeals: **Administrative** appeal and **Medical/Compassionate** appeal.

Reviewer: ```suggestion
There are two types of appeals: **Administrative** appeal and **Medical/Compassionate** appeal.
```
